mtp_api/apps/mtp_auth/tests/test_views.py

```diff
@@ -811,7 +823,12 @@ def assertUserCreated(self, requester, user_data, client_id, groups,  # noqa: N8
             set(new_user.groups.all()),
             set(groups)
         )
-        self.assertSamePrisons(requester, new_user, msg='User Admin able to retrieve users without matching prisons')
+        if assert_prisons_inherited:
+            self.assertSamePrisons(
+                requester, new_user, msg='User Admin able to retrieve users without matching prisons'
+            )
+        else:
+            self.assertNoPrisons(requester, new_user)
```

Comment:
this is checking that the `requester` user has no prisons. should it not be the new user that's verified? something like `self.assertNoPrisons(new_user, msg='New user should have no prisons attached')`

Reply:
Good spot, I hate the way python interprets additional positional arguments as keyword arguements regardless of keyword